src/rt/atomic.rs

```diff
@@ -128,6 +148,10 @@ pub(crate) fn fence(order: Ordering) {
     );
 
     rt::synchronize(|execution| {
+        if execution.log {
+            println!("fence");
```

Comment:
should this include what the ordering of the fence was?

Reply:
Done.